ziti/cmd/ziti/cmd/create_config.go

```diff
@@ -83,6 +83,7 @@ type ControllerTemplateValues struct {
 }
 
 type EdgeControllerValues struct {
+	Port            string
```

Comment:
Port -> AdvertisedPort? also i see `AdvertisedHostPort` still used below. Seems like we need an AdvertisedHost / ListnerHost as well?